lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -4041,33 +4037,133 @@ sealedFee
+        ]
+    , inputs =
+        [ ( TxIn tid 0
+          , TxOut
+                (Address $ unsafeFromHex
+                "714d72cf569a339a18a7d9302313983f56e0d96cd45bdcb1d6512dca6a")
+                (TokenBundle.fromCoin $ Coin 2_000_000)
+          , Just $ Hash $ unsafeFromHex
+            "923918e403bf43c34b4ef6b48eb2ee04babed17320d8d1b9ff9ad086e86f44ec"
+          )
+        ]
+    , redeemers =
+        [ RedeemerSpending (unsafeFromHex "D87A80") (TxIn tid 0)
+        ]
+    }
+  where
+    bytes (Hash x) = x
+    tid = Hash $ B8.replicate 32 '1'
+
+deserializeBabbageTx :: ByteString -> Cardano.Tx Cardano.BabbageEra
+deserializeBabbageTx = either (error . show) id
+    . Cardano.deserialiseFromCBOR (Cardano.AsTx Cardano.AsBabbageEra)
+
+-- Ideally merge with 'updateSealedTx'
+addExtraTxIns
+    :: [TxIn]
+    -> Cardano.Tx Cardano.BabbageEra
+    -> Cardano.Tx Cardano.BabbageEra
+addExtraTxIns ins = modifyBabbageTxBody $ \body ->
+    body
+        { Babbage.inputs = Babbage.inputs body
+            <> (Set.map (Cardano.toShelleyTxIn . toCardanoTxIn) (Set.fromList ins))
+        }
```

Comment:
```suggestion
addExtraTxIns ins = modifyBabbageTxBody $ \body -> body
    { Babbage.inputs = Babbage.inputs body
        <> (Set.map (Cardano.toShelleyTxIn . toCardanoTxIn) (Set.fromList ins))
    }
```

Reply:
I do like keeping `body` visually close to the `{`, but can use a `where` helper: fa0b9b53a